src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+    expect(splitInteger(8, 2)).toEqual([4, 4,]);
```

Comment:
There is an unnecessary trailing comma in the expected array `[4, 4,]`. While this does not affect the test outcome, it is generally good practice to avoid trailing commas in arrays.